src/components/MetricsOptions/MetricsOptionsBar.tsx

```diff
@@ -8,6 +8,7 @@ import { ToolbarDropdown } from '../ToolbarDropdown/ToolbarDropdown';
 interface Props {
```

Comment:
@aljesusg this is exactly what I think should be used instead of React PropTypes

Reply:
Ok , My point is use or no use proptype in all the app and keep all similar. I need feedback about to improve the style code guide